trio/tests/test_util.py

```diff
@@ -53,11 +58,13 @@ async def wait_with_ul1():
 
 def test_module_metadata_is_fixed_up():
     import trio
+
     assert trio.Cancelled.__module__ == "trio"
     assert trio.open_nursery.__module__ == "trio"
     assert trio.abc.Stream.__module__ == "trio.abc"
     assert trio.lowlevel.wait_task_rescheduled.__module__ == "trio.lowlevel"
     import trio.testing
```

Comment:
With the extra blank lines, I think it looks better to put both imports next to each other.